core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -211,7 +211,7 @@ public static String formatPermission(short permission, boolean isDirectory) {
    * @return the JSON-encoded version of the input string
```

Comment:
I think that's a great idea - then it would be a lot easier to keep our error handling and wrapping style consistent